molo/yourwords/tests/fixtures/test.json

```diff
@@ -0,0 +1,18 @@
+[{
```

Comment:
I don't think you need this fixture at all anymore. You can just create a user normally.